src/applications/disability-benefits/2346/components/Batteries.jsx

```diff
@@ -165,8 +151,11 @@ class Batteries extends Component {
             <div
               key={batterySupply.productId}
               className={classnames({
-                'vads-u-background-color--gray-lightest vads-u-padding--3 vads-u-margin-bottom--2': true,
-                'vads-u-border-color--primary vads-u-border--3px': isBatterySelected(
+                'vads-u-background-color--gray-lightest vads-u-margin-bottom--2 vads-u-margin-top--3': true,
+                'vads-u-border-color--primary vads-u-border--3px vads-u-padding--21': isBatterySelected(
```

Comment:
should this be 21? That seems like an odd amount of padding when the design system works in multiples of 4/8 - if this is factored in and is a multiple of 4/8 - thats a lot of padding.  How does this look in responsive when you shrink the view port down?